evalml/automl/automl_search.py

```diff
@@ -816,8 +813,9 @@ def _post_evaluation_callback(self, pipeline, evaluation_results, job_log):
             except PipelineNotFoundError:
                 pass
 
-        if self.search_iteration_plot:
-            self.search_iteration_plot.update()
+        # True when running in a jupyter notebook, else the plot is an instance of plotly.Figure
+        if isinstance(self.search_iteration_plot, SearchIterationPlot):
```

Comment:
Need this check because the api of `SearchIterationPlot.update` and `plotly.Figure` update are now different.

Reply:
Got it, so this line was not covered to begin with right?